R/AllGenerics.R

```diff
@@ -1943,6 +1943,51 @@ setGeneric("stitch", function(object, lockMass, ...) standardGeneric("stitch"))
 setGeneric("stitch.xml", function(object, lockMass) standardGeneric("stitch.xml"))
 setGeneric("stitch.netCDF", function(object, lockMass) standardGeneric("stitch.netCDF"))
 setGeneric("stitch.netCDF.new", function(object, lockMass) standardGeneric("stitch.netCDF.new"))
+
+#' @title Save Xcms objects in a specified format
+#'
+#' @description
+#'
+#' The `storeResults` function saves an `object` resulting from processing with
+#' the `xcms` package (mainly `XcmsExperiment`). Multiple formats for storing
+#' and exporting are available and can be defined by the `param` argument.
+#'
+#' Supported `param` objects are:
+#' 
+#' - `RDataParam`: Save in an .RData format file.
+#' 
+#' - `PlainTextParam`: Save in a plain text format (to be defined).
```

Comment:
Is it fine to add the parameters that are not defined yet ?

Reply:
Yes, that's fine. But please use `[RDataParam()]` instead of `RDataParam` - roxygen will then add a link to the RDataParam documentation (note: you can't that do that now for the `PlainTextParam` because documentation does not yet exist ;) ).